pep-0621.rst

```diff
@@ -38,6 +39,20 @@ The key motivators of this PEP are:
 - Allow for more code sharing between build back-ends for the
   "boring parts" of a project's metadata
 
+To speak specifically to the motivation for static metadata, that has
+been an overall goal of the packaging ecosystem for some time. As
+such, making it easy to specify metadata statically is important. This
+also means that raising the cost of specifying data as dynamic is
+acceptable as users should skew towards wanting to provide static
+metadata.
+
+The push for determinism, pared with static metadata, also helps with
```

Comment:
```suggestion
The push for determinism, paired with static metadata, also helps with
```

? (I'm not sure what this sentence is trying to say, so I may have misinterpreted it, but I doubt that "pared" is correct...)